src/app/modules/rup/components/ejecucion/puntoInicio.html

```diff
@@ -306,39 +306,39 @@
                                         </plex-button>
 
                                         <plex-button size="block"
-                                                     *ngIf="!esFutura(agendaSeleccionada) && agendaSeleccionada.estado !== 'auditada' && turno.estado !== 'suspendido' && turno.paciente && !turno.prestacion && tienePermisos(turno) && verificarAsistencia(turno)"
+                                                     *ngIf="!esFutura(agendaSeleccionada) && agendaSeleccionada.estado !== 'auditada' && turno.estado !== 'suspendido' && turno.paciente && !chequearPrestacion(turno) && tienePermisos(turno) && verificarAsistencia(turno)"
                                                      label="INICIAR PRESTACIÓN" type="success btn-sm"
                                                      (click)="iniciarPrestacion(turno)">
                                         </plex-button>
 
                                         <plex-button size="block"
-                                                     *ngIf="esFutura(agendaSeleccionada) && agendaSeleccionada.estado !== 'auditada' && turno.estado !== 'suspendido' && turno.paciente && !turno.prestacion && verificarAsistencia(turno)"
+                                                     *ngIf="esFutura(agendaSeleccionada) && agendaSeleccionada.estado !== 'auditada' && turno.estado !== 'suspendido' && turno.paciente && !chequearPrestacion(turno) && verificarAsistencia(turno)"
```

Comment:
Ojo con esto de concatenar varios && chequear la guia de desarrollo:

_ngIf: Dentro del if solo se puede poner una condición. Nada de concatenar and con diferentes variables. Si este es el caso se debe llevar a un virtual a la componentes Se recomiendo precalcular el valor en el OnInit o OnChanges, pudiendo así desarrollar mejor la lógica del elemento._

[https://github.com/andes/app/wiki](url)